npm/react/README.md

```diff
@@ -94,14 +88,6 @@ module.exports = (on, config) => {
 
 See [Recipes](./docs/recipes.md) for more examples.
 
-3. ⚠️ Turn the experimental component support on in your `cypress.json`. You can also specify where component spec files are located. For example, to have them located in `src` folder use:
-
-```json
-{
-  "experimentalComponentTesting": true,
-  "componentFolder": "src"
```

Comment:
It's still required, isn't it?

Reply:
It is not required anymore since we do not use preprocessors.
We should update the wizard, I will make a ticket.

Reply:
No `componentFolder` is still required in order to find distinguish component specs and e2e specs.

Reply:
Dmitriy is right.

Reply:
Why isn't it a default? 
I'll fix it